api.py

```diff
@@ -116,5 +117,27 @@ def change_username():
         return_data = {"match": True, "status": 'success', "code": 200}
         return jsonify(return_data)
     else:
-        return_data = {"match": False, "status": 'Unauthorised - incorrect password', "code": 401}
-        return jsonify(return_data)
\ No newline at end of file
+        return_data = {"match": False, "status": "Unauthorised - incorrect password", "code": 401}
+        return jsonify(return_data)
+
+@app.route('/change-avatar', methods=['POST'])
+def change_avatar():
+    data = request.json
+    username = data['username']
+    avatar = data['newAvatar']
+
+    change_avatar_query = update(
+        """UPDATE users 
+        SET dino_id = %s
+        WHERE username = %s""",
+        (avatar, username)
+    )
+
+    if change_avatar_query[1] == 200:
+        return_data = {"match": True, "status": "success", "code": 200}
+        return jsonify(return_data)
+    else:
+        return_data = {"match": False, "status": "internal server error", "code": 500}
+        return jsonify(return_data)
+
```

Comment:
can you put return jsonify(return_data) outside of if statement if ur doing it either way?